packages/react/src/primitives/shared/utils.ts

```diff
@@ -199,9 +199,9 @@ export const classNameModifierByFlag = (
 
 export const getCSSVariableIfValueIsThemeKey = (
   propKey: string,
-  value: string | number
+  value: any
```

Comment:
Should this really be `any`? Maybe we could use a union type of `string` and a generic?

Reply:
Yeah, I think generic makes sense

Reply:
Follow up question not necessarily related to this PR, but does this util need to live in `ui-react`? Looks to be platform agnostic to me

Reply:
how about using `unknown`?

Reply:
Yea, I mean. We can move it to ui package later